<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph App</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="initiateBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Initiate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEdgeBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Edge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nodesTxt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nodes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="weigthTxt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Weigth</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="destinyTxt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Destiny</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sourceTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="startTxt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="endTxt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
